container-metrics.html.md.erb

```diff
@@ -76,17 +76,29 @@ The following table describes all Diego container metrics:
     <td>The log rate limit in bytes per second for an app instance.</td>
     <td><code>float64</code></td>
   </tr>
+  </tr><tr>
+    <td><code>rx_bytes</code></td>
+    <td>Received network traffic in bytes for an app instance.</td>
```

Comment:
Please use `application` instead of the `app` abbreviation everywhere.

Reply:
Thanks for reviewing the docu! :) 

Why should we change ~~`app`~~ to `application`? Throughout this file, `app` is used all the time 🤔

Reply:
"app" is used everywhere, so I don't think that change is necessary. I changed the first instance in the section to "application" in case of possible confusion, but it's true -- "app" is everywhere, in all the doc repos.

Reply:
Developers writing documentation :) It's ok than!